lib/sprinkler_web/live/dashboard_live.ex

```diff
@@ -6,7 +6,12 @@ defmodule SprinklerWeb.DashboardLive do
   @impl true
   def mount(_params, _session, socket) do
     Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry_topic)
-    {:ok, assign(socket, :devices, [%{id: 1, tmps: [5]}, %{id: 2, tmps: [5]}])}
+
+    {:ok,
+     assign(socket, :devices, [
+       %{id: 1, tmps: [nil], hum: [nil], moist: [nil], irrigations: [nil]},
+       %{id: 2, tmps: [nil], hum: [nil], moist: [nil], irrigations: [nil]}
```

Comment:
It might be better to start with an empty array. It's weird to have a `nil` value here when the rest of the values will be numers